### ISSUE TITLE
[Docker] Server not starting (unstable-ssr)

### ISSUE THREAD

**Issue by jakobkukla** [jellyfin/jellyfin-vue#873]
Since unstable-ssr.2021-03-10.71c4d59 (which corresponds to 71c4d59d441ffefbec5382bcbbeee5c459540819) the server is throwing an error on start. The unstable-static tag works fine. (btw what's the difference between these two tags?)

![Screenshot_2021-03-12 Server error](https://user-images.githubusercontent.com/10260599/111006203-aad9e980-838c-11eb-94eb-01a1dee9a3bb.png)

```
ℹ Modern bundles are detected. Modern mode (server) is enabled now.
ℹ Listening on: http://172.23.0.16:80/

 ERROR  Cannot find module 'core-js/modules/esnext.map.delete-all.js' from '/'

  at Function.resolveSync [as sync] (/node_modules/resolve/lib/sync.js:102:15)
  at /node_modules/vue-server-renderer/build.prod.js:1:77685
  at Object.<anonymous> (webpack:/external "core-js/modules/esnext.map.delete-all.js":1:0)
  at __webpack_require__ (webpack/bootstrap:25:0)
  at Object.<anonymous> (server.js:39173:98)
  at __webpack_require__ (webpack/bootstrap:25:0)
  at Module.<anonymous> (server.js:26630:12)
  at __webpack_require__ (webpack/bootstrap:25:0)
  at Object.<anonymous> (server.js:17809:18)
  at __webpack_require__ (webpack/bootstrap:25:0)


 ERROR  Cannot find module 'core-js/modules/esnext.map.delete-all.js' from '/'

  at Function.resolveSync [as sync] (/node_modules/resolve/lib/sync.js:102:15)
  at /node_modules/vue-server-renderer/build.prod.js:1:77685
  at Object.<anonymous> (webpack:/external "core-js/modules/esnext.map.delete-all.js":1:0)
  at __webpack_require__ (webpack/bootstrap:25:0)
  at Object.<anonymous> (server.js:39173:98)
  at __webpack_require__ (webpack/bootstrap:25:0)
  at Module.<anonymous> (server.js:26630:12)
  at __webpack_require__ (webpack/bootstrap:25:0)
  at Object.<anonymous> (server.js:17809:18)
  at __webpack_require__ (webpack/bootstrap:25:0)
```

**Comment by ThibaultNocchi** [comment]
SSR means "server side rendering": in that case, the client is distributed by a Node server which pre compute parts of the app your browser receives. In the classic case, it's a classic HTTP server (nginx in our case) that distributes the client static files. SSR has the benefit of being lighter on the browser at the expense of having to use some resources on the server side of the distribution.

For the issue, @ferferga took it and we should be able to fix it soon enough ;)

**Comment by jakobkukla** [comment]
Thanks for this perfect explanation :)

**Comment by ThibaultNocchi** [comment]
Fixed as of #884 :)
